applications/contrastive_phenotyping/training_script.py

```diff
@@ -233,30 +187,21 @@ def main(hparams):
         num_sanity_val_steps=0,
     )
 
-    # train_loader = data_module.train_dataloader()
-    # example_batch = next(iter(train_loader))
-    # example_input = example_batch[0]
 
-    # wandb_logger.watch(model, log="all", log_graph=(example_input,))
+    print("Trainer initialized!")
 
-    # Fetches batches from the training dataloader,
-    # Calls the training_step method on the model for each batch
-    # Aggregates the losses and performs optimization steps
     trainer.fit(model, datamodule=data_module)
 
     # # Validate the model
     trainer.validate(model, datamodule=data_module)
 
-    # # Test the model
-    # trainer.test(model, datamodule=data_module)
-
 # Argument parser for command-line options
```

Comment:
@alishbaimran In the next PR, this training script should be turned into a config for use with lightning CLI and then deleted.